packages/ingest/src/import-lessons/importLessonsFromCSV.ts

```diff
@@ -60,18 +64,18 @@ export function importLessonsFromCSV({
             };
           });
 
-          await chunkAndPromiseAll({
+          void chunkAndPromiseAll({
             data,
             fn: async (data) => {
               await prisma.ingestLesson.createMany({
                 data,
               });
             },
             chunkSize: 30000,
+          }).then(() => {
+            log.info(`Imported ${parsedLessons.length} lessons from CSV`);
+            resolve();
```

Comment:
What's the error being solved with this change? That the `on` function doesn't accept an async function? The replacement looks like a step backwards to me so it would be useful to know

Reply:
Yes, that's correct. There was a linting complaint about on not accepting an async function

Reply:
Ok, yeah. You can work around it by wrapping it in an arrow function like:

```ts
() => void (async () => {
  ...
})()
```

But that's not too readable!

I'm happy with this to resolve the error